src/toady/reply_service.py

```
@@ -0,0 +1,204 @@
+        else:
+            owner = request.owner
+            repo = request.repo
+
+        # Get pull request number if not provided
+        if not request.pull_number:
+            pull_number = self._get_pull_number_from_comment(request.comment_id)
+        else:
+            pull_number = request.pull_number
+
+        # Construct the API endpoint
+        endpoint = f"repos/{owner}/{repo}/pulls/{pull_number}/comments/{request.comment_id}/replies"
```

Ruff (0.11.9)

Line too long (100 > 88)

(E501)